samples/dnn/segmentation.cpp

```diff
@@ -13,8 +15,8 @@ std::string keys =
     "{ zoo         | models.yml | An optional path to file with preprocessing parameters }"
     "{ device      |  0 | camera device number. }"
     "{ input i     | | Path to input image or video file. Skip this argument to capture frames from a camera. }"
-    "{ framework f | | Optional name of an origin framework of the model. Detect it automatically if it does not set. }"
     "{ classes     | | Optional path to a text file with names of classes. }"
+    "{ num_classes | | number of classes with which the model was trained. If a classes file is provided this parameter will be infered.}"
```

Comment:
In fact, we can just push new colors in case of new class id is more than current number of classes. So please modify code so it can manage it.

Reply:
Hey, @dkurt  I am sorry, but I dont understand what you mean by this, could you please explain with an example. Thanks.